genesyscloud/resource_genesyscloud_journey_segment.go

```diff
@@ -53,7 +53,7 @@ var (
 			}(), ""),
 		},
 		"scope": {
-			Description:  "The target entity that a segment applies to.Valid values: Session, Customer. Changing the scope attribute will cause the existing journey_segment to be dropped and recreated with new ID.",
+			Description:  "The target entity that a segment applies to. Valid values: Session",
```

Comment:
Just correcting the description and leaving the field in place.

Customer scoped segments will be rechartered, and when the time comes, added back here.

No impact to existing customers.